frontend/www/js/omegaup/components/contest/AddContestant.test.ts

```diff
@@ -13,10 +27,47 @@ describe('AddContestant.vue', () => {
           alias: 'Test contest',
           window_length: 20,
         },
-        initialUsers: [],
+        users: [],
       },
     });
 
     expect(wrapper.text()).toContain(T.addUsersMultipleOrSingleUser);
   });
+
+  it('Should handle user list in bulk and in typeahead', async () => {
+    const wrapper = shallowMount(contest_AddContestant, {
+      attachTo: '#root',
+      propsData: {
+        contest: {
+          admission_mode: 'public',
+          alias: 'Test contest',
+          window_length: 20,
+        },
+        users: [],
+      },
+    });
+
+    await wrapper.setData({
+      bulkContestants: 'user, test_user_1',
+      typeaheadContestants: [
+        {
+          key: 'user_1',
+          value: 'User omegaUp',
+        },
+        {
+          key: 'test_user_2',
+          value: 'Test user omegaUp',
+        },
+      ] as types.ListItem[],
+    });
+    await wrapper
+      .find('form button[type="submit"].user-add-bulk')
+      .trigger('click');
+    expect(wrapper.emitted('add-user')).toBeDefined();
+    expect(wrapper.emitted()['add-user']?.[0]).toStrictEqual([
+      ['user', 'test_user_1', 'user_1', 'test_user_2'],
+    ]);
```

Comment:
```suggestion
    expect(wrapper.emitted()['add-user']).toStrictEqual([[
      ['user', 'test_user_1', 'user_1', 'test_user_2'],
    ]]);
```

esto funciona? si no, con el `toEqual` sí debería.

Reply:
Si funciona. Gracias.